redhat-cop/agnosticd
# showroom ocp multiuser, argo and/or helm install

Bastion required
If Argo CD is detected, it is used
If Argo CD is NOT detected, the Helm chart is used as a template.

Each GUID has its own `showroom-GUID` namespace.

If there's multi-user-OCP data in user-data, then showrooms are created `showroom-GUID-userN`

Remove workloads works, too.

- begin work on creating application and appsets
- bump default prod version to chart 0.2.0
- share appset
- significant work. time to test argo
- multi-user argo deployment works
- argocd single and multiuser working
- single and multiple user user_data reporting working
- cleanup some names
- move variable prep to pre_workload so remove_workload will get them.
- get rid of stupid sample data in appset
- put all chart info together
- remove extraneous delete ns
- move most vars to prepare-variables
- add kubeconfig to k8s modules
- remove unnecessary file
- fix kubeconfig indentation
- stop writing user-data.yaml to bastion filesystem
- increate wait time for routes to appear
